protos/escrow/escrow.proto

```diff
@@ -179,6 +204,8 @@ enum EscrowStatus {
   ACTIVE = 1;
   CANCELED = 2;
   CLOSED = 3;
+  NOTSTART = 4;
+  NOTSTART_PAID = 5;
```

Comment:
Is it necessary in the request fields, if the Escrow Status is NOTSTART, it will be also NOTSTART_PAID in the receiver.

Reply:
NotStart refer to the status the escrow receive the renew contracts, NotStart_paid refer to the status escrow receive the payin after the NotStart. Such status only can be read by the other system.